akka-actor/src/main/scala/akka/actor/Timers.scala

```diff
@@ -103,6 +113,15 @@ abstract class AbstractActorWithTimers extends AbstractActor with Timers {
    */
   def startSingleTimer(key: Any, msg: Any, timeout: FiniteDuration): Unit
 
+  /**
+   * Same as [[startSingleTimer]],
```

Comment:
I more prefer `copy and paste`, without a second click to see all.

Reply:
I don't really understand the point here. Could you be more explicit?

Reply:
Instead of referring to the other `startSingleTimer` here, it might be better to just copy-and-paste the scaladocs. While this is a greater maintenance burden, it is much more pleasant for users reading the docs.

Reply:
i get it

Reply:
👍 all places

Reply:
@patriknw please waht do you mean with all places? Do you mean no only Timers#startSingleTimer but in all methods in JavaDSL?

Reply:
Yes, please copy the documentation of all the new merhods that are added in this PR